### ISSUE TITLE
Feature Request: Integrated image upscale

### ISSUE THREAD

**Issue by lonnylippold** [divamgupta/diffusionbee-stable-diffusion-ui#120]
Would be nice to have an integrated image upscale with aspect ratio setting

**Comment by ideologysec** [comment]
RealESRGAN might be the best candidate for this, there are several Stable Diffusion repos out there that include it and GFPGAN

invoke-ai (formerly lstein): https://github.com/invoke-ai/InvokeAI
sd-webui (formerly hlky): https://github.com/sd-webui/stable-diffusion-webui

**Comment by greggh** [comment]
For now there is a great app that uses RealESRGAN and RealSR. I use Upscayl for this after generating with DiffusionBee

https://github.com/upscayl/upscayl

**Comment by lonnylippold** [comment]
Yes I like Upscale as well. I also use https://github.com/NightmareAI/Real-ESRGAN via script
